templates/steps/ado-configure.yaml

```diff
@@ -35,6 +35,10 @@ input:
       - Defra-ADP-Github
       - DEFRA-ADP-SonarCloud
       - DEFRA-ADP
+      - AZD-ADP-AdoCallbackApi-DEV
+      - AZD-ADP-AdoCallbackApi-TST
+      - AZD-ADP-AdoCallbackApi-PRE
```

Comment:
PRE  connection name should be renamed as AZR-ADP-AdoCallbackApi-PRE
PPRD connection name should be renamed as AZR-ADP-AdoCallbackApi-PRD

Reply:
Theyre both AZD on the ticket in ADO, happy to change them if needed

Reply:
Renamed